### ISSUE TITLE
Warnings are not sent in cql response

### ISSUE THREAD

**Issue by soyacz** [scylladb/scylla#10196]
*Installation details*
Scylla version (or git commit hash): 4.6.rc5

When using Cassandra, python cql client response contain warnings created by server. 
When using Scylla, warnings are not sent - only place to see them is scylla log.

**Comment by nyh** [comment]
> When using Cassandra, python cql client response contain warnings created by server. When using Scylla, warnings are not sent - only place to see them is scylla log.

It sounds like you're making in this issue a general claim that Scylla doesn't know how to send warnings in CQL responses, and yet your patch seems to suggest that this is only the case in one specific case (an over-sized batch). If it's just this specific case, the issue should probably be renamed.

**Comment by soyacz** [comment]
> > When using Cassandra, python cql client response contain warnings created by server. When using Scylla, warnings are not sent - only place to see them is scylla log.
> 
> It sounds like you're making in this issue a general claim that Scylla doesn't know how to send warnings in CQL responses, and yet your patch seems to suggest that this is only the case in one specific case (an over-sized batch). If it's just this specific case, the issue should probably be renamed.

Yes, my mistake. I cannot make general statements from specific scenario. Thanks for pointing that.

**Comment by nyh** [comment]
We have a cql-pytest test reproducing this issue, `test_batch.py`::`test_warnings_are_returned_in_response`.

**Comment by nyh** [comment]
I added the "guardrails" tag because it checks the `batch_size_warn_threshold_in_kb` and shows it doesn't **warn** properly. The other half of the guardrail, `batch_size_fail_threshold_in_kb`, works as expected (as other tests in the same file show).

By the way, Cassandra renamed these two configuration parameters - it removes the "in_kb" part from the name, and now the value should include units (e.g., "50KiB"). But that is a separate issue.
